commands/utils/params.go

```diff
@@ -29,13 +29,29 @@ var errFrogbotConfigNotFound = fmt.Errorf("%s wasn't found in the Frogbot direct
 var osFrogbotConfigPath = filepath.Join(frogbotConfigDir, FrogbotConfigFile)
 
 type FrogbotUtils struct {
-	ConfigAggregator *FrogbotConfigAggregator
+	ConfigAggregator FrogbotConfigAggregator
 	ServerDetails    *coreconfig.ServerDetails
 	Client           vcsclient.VcsClient
 }
 
 type FrogbotConfigAggregator []FrogbotRepoConfig
 
+// UnmarshalYaml uses the yaml.Unmarshaler interface to parse the yamlContent, and then sets default values if it wasn't supplied by the user.
```

Comment:
...if it wasn't supplied... --> ...if they weren't set...

Reply:
done